### PR TITLE
Minor cleanups in crowbar cookbook + add crowbar.service systemd file for SLE12

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: chef/cookbooks/crowbar/recipes/default.rb [hunk 1]
```diff
@@ -352,23 +339,60 @@
...
+      mode "0644"
+      action :create
+      source "crowbar.service"
+    end
+
+    # Make sure that any dependency change is taken into account
+    bash "reload systemd after crowbar update" do
+      code "systemctl daemon-reload"
+      action :nothing
+      subscribes :run, resources("cookbook_file[/etc/systemd/system/crowbar.service]"), :immediately
```
Comment (vuntz) [inline chef/cookbooks/crowbar/recipes/default.rb]: @tboerger what would be the proper way of writing this line in multiline? Just go back to a new line after each `,`?

Comment (tboerger) [inline chef/cookbooks/crowbar/recipes/default.rb]: Yes, something like that.

**Comment by tboerger** [comment]
Some tiny Hound offense but LGTM

**Comment by rsalevsky** [comment]
@tboerger The new SUSE style says Metrics/LineLength Max 100.

:+1:

File: chef/cookbooks/crowbar/recipes/default.rb [hunk 2]
```diff
@@ -352,23 +339,60 @@
...
-    action :create
-    to "/etc/init.d/crowbar"
-    not_if "test -L /usr/sbin/rccrowbar"
+    link "/usr/sbin/rccrowbar" do
+      action :create
+      to "/etc/init.d/crowbar"
+    end
+
+    # Make sure that any dependency change is taken into account
+    bash "insserv crowbar service" do
```
Comment (tboerger) [inline chef/cookbooks/crowbar/recipes/default.rb]: This is already in service enable implied?

Comment (vuntz) [inline chef/cookbooks/crowbar/recipes/default.rb]: Yes when the service is not enabled, but if it's already enabled and the init file changes, then we need to call insserv to update the dependency chain.

Comment (tboerger) [inline chef/cookbooks/crowbar/recipes/default.rb]: Ok

**Comment by tboerger** [comment]
It's not beautiful but it's the case for that cookbook anyway :D
:+1:

**Pull request merged**